Fix NextJS example (#3540)
SAP/luigi: core/examples/luigi-example-next/README.md

Due to the nature of NextJS's routing strategy, it is not possible to use `useHashRouting: false` in the Luigi Configuration. If you want to use path based routing, Luigi Core's index.html should be served separately.

> Also note that due to NextJS's server side nature, injecting LuigiClient in your pages is not as straightforward, since LugiClient relies on the window object only available at client side. To resolve that simply include the luigi-client.js as the source of an HTML script as shown in the home.js file. 
> This will allow to use LuigiClient as part of the window object of that microfrontend ex: `window['LuigiClient']`

Reviewer: ```suggestion
>**NOTE**: Due to NextJS's server side nature, injecting Luigi Client in your pages is not as straightforward, since Lugi Client relies on the window object only available at the client side. To resolve that, simply include the `luigi-client.js` file as the source of an HTML script as shown in the `home.js` file. 
This will allow you to use Luigi Client as part of the window object of that micro frontend, for example: `window['LuigiClient']`
```

npm install
```

> Second, build the next app:

Reviewer: ```suggestion
Second, build the NextJS app:
```

```bash
npm run build
```

> Next run the server:

Reviewer: ```suggestion
Next, run the server:
```
